### PR TITLE
vmods

### PR DESCRIPTION
**Pull request by mamico**

### CODE REVIEW

File: plone/recipe/varnish/recipe.py
Comment (jensens) [inline plone/recipe/varnish/recipe.py:84]: Is it by intend to first call super build here and again in the conditional block? 
If so please add a comment why, because it is not obvious

Comment (cleberjsantos) [reply plone/recipe/varnish/recipe.py:84]: +1

Comment (mamico) [reply plone/recipe/varnish/recipe.py:85]: Agree... reimplemented in a more clear manner.

**Review by fredvd — APPROVED** [review]
LGTM, I'll test once merged. 

I tried to add compiling 'generic' vmods on a local branch year but got my self into trouble so quickly I gave up. Most of them need the varnish source installed and configured first.  

I assume that works at this position because you first call super and then do another cmmi round.  Could we also add other vmod archives here later (if we find a sane way to add them in buildout options... :-O

**Comment by mamico** [comment]
@fredvd before this PR, I used following recipes for building vmods, probably could be better to suggest something like that instead of doing strange things here...

```
[varnish-modules]
recipe = zc.recipe.cmmi
url = https://github.com/varnish/varnish-modules/archive/0.15.1.tar.gz
source-directory-contains = bootstrap
configure-command = ./bootstrap; ./configure
environment =
    PKG_CONFIG_PATH=${varnish-build:location}/lib/pkgconfig

[varnish-modules-copy]
recipe = collective.recipe.cmd
on_install = true
on_update = true
cmds =
    cp ${varnish-modules:location}/lib/varnish/vmods/* ${varnish-build:location}/lib/varnish/vmods/

```

**Pull request merged**
